doc/cargo-doc-live.md

```diff
@@ -0,0 +1,40 @@
+# cargo-doc-live
+
+[cargo-doc-live] is live server version of `cargo doc`.
+
+{#start}
+
+## Getting started
+
+```nix
+# In `perSystem.process-compose.<name>`
+{
+  services.cargo-doc-live."cargo-doc-live1".enable = true;
+}
+```
+
+{#port}
+
+### The port for `cargo doc`
+
+```nix
+{
+  services.cargo-doc-live."cargo-doc-live1" = {
+    enable = true;
+    port = 8080;
+  };
+}
+```
+
+{#crateName}
+
+### The crate to use when opening docs in browser
```

Comment:
I know our docs elsewhere are more of skeleton, but we should get into the practice of fleshing out when writing *new* docs. Can you add some paragraphs to all these sections?